cli/lib/kontena/debug_instrumentor.rb

```diff
@@ -52,14 +53,14 @@ def self.instrument(name, params = {}, &block)
         result << str
       end
 
-      if $stderr.tty?
+      if Kontena.logger.instance_variable_get(:@logdev).dev.tty?
```

Comment:
Ew :D Would be it better to just keep the `DebugInstrumentor` hardcoded to use $stderr?